### PR TITLE
Remove ExtraField

### PR DESCRIPTION
**Pull request by symroe**
The "ExtraField" concept allowed site admins to add fields in the Django admin and have those fields show in on the Person model. 

It was a nice idea, but made things complex. In our case we can just add or remove fields at the model level, and start using the forms framework to update models in the frontend.

This should be seen more as removing a feature than improving the ability to add new fields – the frontend Person CRUD is still quite complex, and dynamically created in the forms `__init__` method (and others).

Removing ExtraFields is one step on the way to making things simpler though. 

Also, the single thing we were using this whole feature for was the biscuit field. That's a lot of code for a tiny bit of fun. I've moved it to the Person model (assuming it's not a property of the candidacy!)

### CODE REVIEW

**Comment by symroe** [comment]
I've also split a lot of the "delete stuff" commits in to abc6116 72d1691 and 484c965. There's no _real_ reason for this, apart from it made picking out the edits a little easier (they're per app) and it might make reviewing a bit easier.

Also, remember this is based on to the `person_model_changes` branch. I think this might be the last one on to that branch before merging and deploying. 

I'd like to get that done this year if at all poss…

File: ynr/apps/elections/uk/templates/candidates/person-view.html [hunk 1]
```diff
@@ -201,13 +201,11 @@ <h2>{% trans "Demographics:" %}</h2>
     </dl>
   {% endif %}
 
-  {% if extra_fields %}
+  {% if person.favourite_biscuit %}
```
Comment (chris48s) [inline ynr/apps/elections/uk/templates/candidates/person-view.html]: Do we want to show/hide this entire "Additional information" based on presence/absence of biscuit? Now that its a "core" field, should we just always show it regardless of whether its populated?

Comment (symroe) [reply ynr/apps/elections/uk/templates/candidates/person-view.html]: Yeah, clearly this isn't the long term solution, but this does keep the existing functionality as biscuits are the only ExtraField. 

Happy to move it up the page to the person info if you think that's useful?

Comment (chris48s) [reply ynr/apps/elections/uk/templates/candidates/person-view.html]: Meh - if it makes more sense to think about it under the banner or redesigning the "person" page we can leave it for now and do it then.

Comment (symroe) [reply ynr/apps/elections/uk/templates/candidates/person-view.html]: Yeah, I'll leave that for a later refactor of this page

**Comment by symroe** [comment]
Thanks 👍

**Pull request merged**